subsys/shell/shell_utils.c

```diff
@@ -359,12 +359,8 @@ static void buffer_trim(char *buff, u16_t *buff_len)
 	/* Counting whitespace characters starting from beginning of the
 	 * command.
 	 */
-	while (isspace((int) buff[i++])) {
-		if (i == 0) {
```

Comment:
This was implemented to prevent `i` overflow.

Reply:
Ok. What time "i" can be 0? While loop runs for the first time and "i" is already 1.

Reply:
When loop will run 65536 times. This is taken over from Nordic SDK where buff len could be u8, u16 or u32.
However in this case it is quite unlikely we will have a command composed of more than 65536 characters :)

Reply:
Yes, right, my fault:(
But what we can do with that Coverity issue? Just square it off?

Reply:
It's so unlikely that we will have an overflow for u16 that we can keep this change :)